health.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+	"time"
+)
+
+var lastTrigger int64
+
+func startRestAPI() {
+	lastTrigger = time.Now().Unix()
+	router := mux.NewRouter()
+	router.Methods("GET").Path("/health").HandlerFunc(getHealth)
+	router.Use(errorLogger)
+	go func() {
+		log.Fatal(http.ListenAndServe(":7070", router))
+	}()
+}
+
+func triggerHealth() {
+	lastTrigger = time.Now().Unix()
+}
+
+func ishealthy() bool {
+	return time.Now().Unix()-lastTrigger < int64(brf.Polltime*2)
```

Comment:
https://godoc.org/time
It's idiomatic in go to use the time package, and not convert back and forth to unix timestamps all the time.
something like:
```
type healthCheck struct {
sync.Mutex
lastPing time.Time
}
func (hc *healthCheck) ping() {
    hc.Lock()
    defer hc.Unlock()
    hc.last = time.Now()
}
func (hc *healthCheck) isHealthy() bool {
    hc.Lock()
   defer hc.Unlock()
   return time.Since(hc.last) < 2*time.Minute
}

var health := &healthCheck{}
```